domains/fqdn.go

```diff
@@ -0,0 +1,25 @@
+package domains
+
+import (
+	"fmt"
+	"regexp"
+)
+
+type FQDN struct {
+	Name string `json:"name"`
+	Tld  string `json:"tld"`
+}
+
+// Validate the FQDN struct to make sure the name conforms to the expected format
```

Comment:
What do you think if we stick to [this](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences) style guide for comments?
It would be something like:
```suggestion
// Validate makes sure that the name of the FQDN struct conforms to the expected format.
```
but feel free to adjust as you prefer.

Reply:
I updated the comment per your suggestion, and added a comment to the other Validate method.